src/components/Article.jsx

```diff
@@ -1,8 +1,21 @@
+import { useState } from "react";
+import LikeButton from "./LikeButton";
 import "./style/articleView.css";
+import { updateArticleVotes } from "./utilities/requests";
 
-export default function Article(props) {
-  const article = props.children;
+export default function Article({ article, setArticle }) {
   const dateCreated = new Date(article.created_at);
+  const [hasVoted, setHasVoted] = useState(false);
+
+  const handleVote = () => {
+    if (!hasVoted) {
+      updateArticleVotes(article.article_id, 1).catch(() => {
```

Comment:
You need a `.then` in here before a `.catch` so you can control what you get back from the server